src/graph.rs

```diff
@@ -532,12 +532,30 @@ impl Serialize for Value {
             Value::Boolean(value) => serializer.serialize_bool(*value),
             Value::Integer(value) => serializer.serialize_u32(*value),
             Value::String(value) => serializer.serialize_str(value),
```

Comment:
Maybe we should serialize these as hashes too?

Reply:
Uniformity, basically: serializing all of the different types of `Value` the same way. I rephrase the question as: how do we value uniformity of `Value` serializations relative to sticking close to JSON's primitives where possible?